pkg/commands/compute/deploy.go

```diff
@@ -839,7 +839,7 @@ func pkgUpload(spinner text.Spinner, client api.Interface, serviceID string, ver
 	return spinner.Stop()
 }
 
-func constructSetupObjects(
+func constructSetupKVs(
```

Comment:
Wondering if this should have been left as `constructSetupObjects`? Since `setup.Backends`, for example, aren't key-value stores. Working through merge conflicts in #769 and noticed this.

Reply:
@awilliams-fastly correct! I'm actually fixing this at the moment 🙂